Django-CoWork-Connect/coapp/admin.py

```diff
@@ -1,8 +1,30 @@
 from django.contrib import admin
-from .models import Desk
+from .models import Desk, Reservation
+
 
-@admin.register(Desk)
 class DeskAdmin(admin.ModelAdmin):
     list_display = ('stock_number', 'size', 'monitor_number', 'power_socket_count', 'price', 'status')
     list_filter = ('monitor_number', 'size', 'status')
-    search_fields = ['stock_number']
\ No newline at end of file
+    search_fields = ['stock_number', 'status']
+    actions = ['cancel_reservations']
+
+
+class ReservationAdmin(admin.ModelAdmin):
+    list_display = ('desk', 'id', 'user', 'start_date', 'end_date', 'total_cost')
+    list_filter = ('start_date', 'end_date', 'user')
+    search_fields = ('id', 'user__username', 'desk__name')
+    raw_id_fields = ('desk', 'user')
+
+    def cancel_reservations(self, request, queryset):
+        for reservation in queryset:
+            desk = reservation.desk
```

Comment:
można to zrobic bez petli i dwoama zapytaniami do bazy. Mniej wiecej tak 
```suggestion
            desks_ids = quryset.values_list("desk_id")
            Desks.objects.filter(id__in=desks_ids).update(status="czynne")
            queryset.delete()
```

Reply:
popracuję nad tym